src/index.js

```diff
@@ -1,15 +1,26 @@
 const arrify = require('arrify');
 const merge = require('deepmerge');
+const path = require('path');
 
 module.exports = ({ config }, options) => {
+  const NODE_MODULES = path.join(__dirname, 'node_modules');
+  const LOADER_EXTENSIONS = /\.vue$/;
+  const styleRule = config.module.rules.get('style');
+  const stylePostCssLoader = styleRule && styleRule.uses.get('postcss');
+  const lintRule = config.module.rules.get('lint');
+
   config.module.rule('vue')
-    .test(/\.vue$/)
+    .test(LOADER_EXTENSIONS)
     .use('vue')
-    .loader(require.resolve('vue-loader'))
-    .options(options);
+      .loader(require.resolve('vue-loader'))
+      .options(options);
+
+  config.resolve.extensions.add('.vue');
```

Comment:
I understand. It seems like the `webpack` template for `vue-cli` actually uses it, so I guess it might be a common pattern. I'll make sure to merge it.